streams/src/main/java/org/apache/kafka/streams/kstream/JoinWindows.java

```diff
@@ -103,6 +103,11 @@ public JoinWindows after(long timeDifference) {
     }
 
     @Override
+    public long size() {
```

Comment:
Should we throw an `UnsupportedOperationException` for join windows since it should not be called?